docs/sections/user_guide/cli/drivers/index.rst

```diff
@@ -4,5 +4,6 @@ Drivers
 .. toctree::
    :maxdepth: 1
 
+   chgres_cube
```

Comment:
chgres_cube --> Mode chgres_cube

Reply:
This one was updated in PR #429, so I went ahead and preemptively made the change before that merged.